vud.theme.inc

```diff
@@ -101,7 +124,7 @@ function vud_pseudo_theming($vote_entity, $template_type, $plugin, &$variables)
   if ($function = ctools_plugin_get_function($plugin, 'alter template variables')) {
     $function($template_type, $variables);
   }
-
+  
```

Comment:
Spaces without a real line are against coding standards, please remove this unneeded and wrong change.